blob/src/test/java/io/crate/integrationtests/BlobIntegrationTest.java

```diff
@@ -305,4 +310,37 @@ public void testIndexOnNonBlobTable() throws IOException {
             is("{\"_index\":\"test_no_blobs\",\"_type\":\"default\"," +
                "\"_id\":\"1\",\"_version\":1,\"_shards\":{\"total\":1,\"successful\":1,\"failed\":0},\"created\":true}"));
     }
+
+    @Test
+    public void testBlobShardStatsAreUpdatedOnUploadAndDelete() throws IOException {
```

Comment:
Uploading blobs with the same digest results in 409
 I'm adding another delete to make sure the stats are updated only once.